winpr/libwinpr/synch/synch.h

```diff
@@ -32,6 +32,10 @@
 #define WITH_POSIX_TIMER	1
 #endif
 
+#ifdef __sun
```

Comment:
Maybe we should move this test to a CMake script checking for the necessary API, so we don't have to do this for every platform.

Reply:
Created #4608 to fix the timer issue, please remove here.